apps/andi/lib/andi_web/live/edit_live_view.ex

```diff
@@ -1,5 +1,7 @@
 defmodule AndiWeb.EditLiveView do
   use AndiWeb, :live_view
+  use AndiWeb.HeaderLiveView,
+    prompt_for_changes?: true
```

Comment:
Don't we want this to be false by default? If I am reading this right, this will prompt the user about unsaved changes even if they don't change anything and they click a link in the header

Reply:
Oh oops, maybe I should call it prompt_on_changes or something like that. It is false by default as the list pages don't have changes